cunumeric/random/generator.py

```diff
@@ -364,31 +394,29 @@ def zipf(
         return self.bit_generator.zipf(alpha=a, shape=size, dtype=dtype)
 
 
-# TODO: Leaving the type of `seed` vague, since the proper type
-# `Union[None, int, BitGenerator, Generator]` currently breaks the docs build.
```

Comment:
Please don't remove this TODO if you're not going to update the type signature.

Reply:
af68e77cf1c8f1d90456da9368f34d9ca378a806